experimental/components/Cargo.toml

```diff
@@ -46,29 +46,88 @@ log = { version = "0.4", optional = true }
 num-bigint = "0.4.4"
 num-rational = "0.4.1"
 num-traits = "0.2.17"
-serde = { version = "1.0", default-features = false, features = ["derive", "alloc"], optional = true }
+serde = { version = "1.0", default-features = false, features = [
+    "derive",
+    "alloc",
+], optional = true }
 smallvec = "1.11.2"
 
 icu_experimental_data = { workspace = true, optional = true }
-icu_locid_transform = { workspace = true, optional = true, features = ["compiled_data"] }
+icu_locid_transform = { workspace = true, optional = true, features = [
+    "compiled_data",
+] }
 
 [target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
 criterion = "0.4.0"
 
 [dev-dependencies]
 icu_locid = { path = "../../components/locid" }
+num = "0.4.1"
```

Comment:
do not pull in the `num` crate.

Reply:
this is `[deb-dependencies]` ?

Reply:
you're not even using it

Reply:
I use it in `use num_traits::sign::Signed;` because I use `abs()` fn.

Reply:
In `use num_traits::sign::Signed;` you use `num_traits`